spec/services/admin/organizations/update_service_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::Organizations::UpdateService do
+  subject(:update_service) { described_class.new(organization:, params:) }
+
+  let(:organization) { create(:organization) }
+
+  let(:params) do
+    {
+      name: 'FooBar',
+    }
+  end
+
+  describe '#call' do
+    it 'updates the organization' do
+      result = update_service.call
+
+      expect(result.organization.name).to eq('FooBar')
```

Comment:
expect { update_service.call }.to change { organization.reload.name }.to('FooBar')